csuntechlab/iSTART
# STR-161 -- UI: Slides continue button should indicate when it's clickable on mobile

# Description
Fixed up bottom navbar; Added tooltip; Fixed email placeholder.
  
# Testing
1. Login as `cyndee.bellamy` or `april.feldman`.
2. Visit the ***Alcohol Module***.
3. Go through the slides and make sure on `Desktop` the bottom `navbar` isn't broken visually and all the previous functionality works.
4. For each slide, make sure there is a proper tooltip when you ***click*** or ***hover over*** the continue button.
5. Repeat ***`step3`*** & ***`step4`*** on mobile.

### STR-176 & STR-179 Accomplished in this task
Please check and move both `STR-176` & `STR-179` after you go through STR-161.
